examples/python/utilities.py

```diff
@@ -0,0 +1,58 @@
+from pyspark import SparkContext
+from pyspark.sql import Column, DataFrame, SparkSession, SQLContext, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import threading
+
+# Create SparkContext
+sc = SparkContext()
+sqlContext = SQLContext(sc)
+
+# Enable SQL for the current spark session.
+spark = SparkSession \
+    .builder \
+    .appName("...") \
+    .master("...") \
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
+    .config("spark.sql.sources.parallelPartitionDiscovery.parallelism","8") \
```

Comment:
nice! add comment above each conf saying why the two confs are needed 

io.delta.sql.DeltaSparkSessionExtension -- to enable custom Delta-specific SQL commands
parallelPartitionDiscovery.parallelism -- control the parallelism for vacuum